### ISSUE TITLE
Feature request: Column Align

### ISSUE THREAD

**Issue by amirm** [hltj/wxMEdit#70]
Delete spaces and TABs at right of column selection until EndOfLine or CHAR.

Example: Using column mode, select a line in the text:
foo, |         bar
foo, |     bar
foo, |       bar
foo, |         bar
foo, |        bar

Edit, Advanced, Column Align
foo, bar
foo, bar
foo, bar
foo, bar
foo, bar

Thanks in advance

**Comment by amirm** [comment]
Greetings!

I saw that this feaure is scheduled for version 3.0.

I just realized that it almost exists - if you press CTRL-DEL wxMEdit wlll delete SPACE and TAB from current cursor position to first non-{SPACE/TAB} of line.

All wxMEdit has to do is apply this to each line selected in column mode.

Yes, it was me who posted the "The same thing as http://code.google.com/p/wxmedit/issues/detail?id=4"

Once again, thank you very much for your effort. 
I (and hopefully, all other users of wxMEdit) appreciate it.
